src/lib.rs

```diff
@@ -249,7 +275,7 @@ macro_rules! impl_bounded_map {
                 {
                     self.0
                         .get(&TypeId::of::<T>())
-                        .and_then(|any| any.downcast_ref())
+                        .and_then(|any| (&**any as &(dyn Any)).downcast_ref())
```

Comment:
Why ?

Reply:
any is potentially a Box<dyn Any + Send + Sync> and downcast_ref is not available in that case